src/index.html

```diff
@@ -9,5 +9,13 @@
   </head>
   <body>
     <h1>Stars</h1>
+    <div class="container">
+      <div class="star--0"></div>
+      <div class="star--1"></div>
+      <div class="star--2"></div>
+      <div class="star--3"></div>
+      <div class="star--4"></div>
+      <div class="star--5"></div>
```

Comment:
In the terms of task there is point like

![image](https://user-images.githubusercontent.com/122989585/215124471-d5d7ee87-5cab-42a3-aeb2-025a62e8d793.png)

So you should create root div element for each row of stars and place the stars inside:
```suggestion
      <div class="stars stars--0">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
      </div>
      
      <div class="stars stars--1">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
      </div>
      
      ... and so on
```